### PR TITLE
refactor: depend on d3-array and d3fc-rebind

### PR DESCRIPTION
**Pull request by tlsim**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,31 +17,35 @@
...
   "devDependencies": {
     "babel-preset-es2015-rollup": "^1.1.1",
-    "d3fc": "^7.0.0",
+    "browserify": "^13.0.1",
+    "d3": "^3.5.17",
     "eslint": "^2.2.0",
     "eslint-config-airbnb": "^6.0.2",
     "eslint-config-standard": "^5.1.0",
     "eslint-plugin-promise": "^1.1.0",
+    "eslint-plugin-react": "^4.2.3",
```
Comment (ColinEberhardt) [inline package.json]: Just wondering why we need a react eslint plugin?

**Comment by ColinEberhardt** [comment]
Nice job - thanks. Just one minor question

File: package.json (hunk 1 above)
Comment (tlsim) [inline package.json]: I get a warning from npm without it:
eslint-config-airbnb@6.2.0 requires a peer of eslint-plugin-react@^4.2.3 but none was installed.

Comment (ColinEberhardt) [inline package.json]: Fair enough - just checking it was needed. I have a terrible habit of leaving un-necessary stuff in package.json!

**Pull request merged**